cmd/rest-server/main.go

```diff
@@ -47,6 +47,7 @@ func init() {
 	flags.StringVar(&server.TLSCert, "tls-cert", server.TLSCert, "TLS certificate path")
 	flags.StringVar(&server.TLSKey, "tls-key", server.TLSKey, "TLS key path")
 	flags.BoolVar(&server.NoAuth, "no-auth", server.NoAuth, "disable .htpasswd authentication")
+	flags.StringVar(&server.HtpasswdPath, "htpasswd-file", server.HtpasswdPath, "location of .htpasswd file (default: \"<data directory>/.htpasswd)\"")
```

Comment:
My intention was to make it clear that this argument was expecting the path to the file and not, say, a directory with a `.htpasswd` file. I also didn't want the variable name to be confused with the `htpasswdFile *HtpasswdFile` variable already present on `Server`. I would be glad to change it to whatever you think would be most clear.

Reply:
You mean that your intention with the `--htpasswd-file` name was this, I take it? As in, the `HtpasswdPath` is what you think makes the most sense to have in the code, but for the user interface you opted to use `-file` instead of `-path` in the option name? I don't disagree with that, I just think that we generally keep them the same name. But let's see what @MichaelEischer or @fd0 says about it!

Reply:
> You mean that your intention with the --htpasswd-file name was this, I take it?

That's correct. 

Another option would be something along the lines of `--htpasswd-filepath` (and `HtpasswdFilepath`) or some alternative.

Reply:
Yeah, but that's too long :) Honestly I think the user facing option name is fine as it is now, because it's in line with `restic --password-file`. And for that option restic uses `PasswordFile`, hence my thinking of coherency. But I'll leave this PR for further review by the ones I mentioned above anyway, so let's see what they say about renaming this so it's like in restic's code. Personally I would rename it. But it's right and wrong however you turn it around.

Reply:
The variable name `HtpasswdFile` has the problem that there's also a struct member `htpasswdFile`, which could be rather confusing. As a middle ground we could name the variable `HtpasswdFilepath` but keep the option names as is. But I also don't mind to keep the current name.

Reply:
@rawtaz any update on a preferred name for this variable? All the remaining suggestions/comments should be incorporated now.